File: 2nd_Algo/taeyun/1546.py
N = int(input())
scores = list(map(int, input().split()))

# 세준이의 최고점을 찾음
max_score = max(scores)

# 각 점수를 새로운 점수로 변환하여 총합을 구함
new_sum = sum(score / max_score * 100 for score in scores)

# 평균을 구하고 출력
print(new_sum / N)

File: 2nd_Algo/Hobeom/1546.c
#include <stdio.h>


int main() {

	double jumsu[1001];
	int t, max = -1;
	double sum = 0;

	scanf("%d", &t);

	for (int i = 0; i < t; i++) {
		scanf("%lf", &jumsu[i]);
		if (max < jumsu[i]) max = jumsu[i];
	}

	for (int i = 0; i < t; i++) {
		jumsu[i] = jumsu[i] / max * 100;
		sum += jumsu[i];
	}

	printf("%lf", sum / t);



	return 0;
}
